### ISSUE TITLE
Keyboard loses focus during login with wifi enabled

### ISSUE THREAD

**Issue by bamarco** [pop-os/gdm3#23]
Sometimes when logging in I will be halfway through typing my password and the login box will lose keyboard focus to the network manager. I am not logged in so the network manager should not be able to take keyboard focus. This will cause only half of my password to be entered causing a failed login. This only occurs when I am logging in to the laptop after freshly entering the zone of a known wifi network (possibly only wifi with captive login pages)
